### PR TITLE
Add borderRadius to NavLink

### PR DESCRIPTION
**Pull request by lukebennett88**

### CODE REVIEW

**Review by JedWatson — APPROVED** [review]

**Review by JedWatson — CHANGES_REQUESTED** [review]
Looks good, needs a changeset

**Comment by lukebennett88** [comment]
> Looks good, needs a changeset

Good call. Updated.

File: .changeset/hot-yaks-hide.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@spark-web/nav-link': patch
```
Comment (JedWatson) [inline .changeset/hot-yaks-hide.md:2]:
```suggestion
'@spark-web/nav-link': minor
```

**Review by JedWatson — CHANGES_REQUESTED** [review]
This adds a feature, so should be `minor`

**Review by JedWatson — APPROVED** [review]

**Pull request merged**